### PR TITLE
Initial sched config.

### PR DESCRIPTION
**Pull request by jasonbs10**

### CODE REVIEW

**Comment by dbentley** [comment]
LGTM

I think this'll evolve, but it'd be good to unblock for now...

File: sched/config/config.go [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+	"github.com/apache/thrift/lib/go/thrift"
+	clusterdef "github.com/scootdev/scoot/cloud/cluster"
+	clusterimpl "github.com/scootdev/scoot/cloud/cluster/memory"
+	"github.com/scootdev/scoot/saga"
+	"github.com/scootdev/scoot/sched/distributor"
+	"github.com/scootdev/scoot/sched/queue"
+	queueimpl "github.com/scootdev/scoot/sched/queue/memory"
+	"github.com/scootdev/scoot/sched/scheduler"
+	"github.com/scootdev/scoot/sched/worker"
+	//"github.com/scootdev/scoot/sched/worker"
```
Comment (CaitieM20) [inline sched/config/config.go]: nit: remove commented out imports

**Comment by CaitieM20** [comment]
LGTM.

One future thinking thing that we may want to add to the backlog.  In prod we probably never want to use the default plugins.  Signals a configuration error.  We could add a flag to the aurora config based on what environment its deployed in and disallow the use of plugins in prod.

**Comment by CaitieM20** [comment]
One more note can we add a test case to prove the config files parse correctly.  I've found this super valuable in the past

**Pull request merged**